modules/metering-uninstall-crds.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-uninstall.adoc
+
+[id="metering-uninstall-crds_{context}"]
+= Uninstalling metering CustomResourceDefinitions
+
+The metering CustomResourceDefinitions (CRDs) remain in the cluster after the Metering
+Operator is uninstalled and the `openshift-metering` namespace is deleted.
+
+[IMPORTANT]
+====
+Deleting the metering CRDs disrupts any additional metering installations in other namespaces.
+Ensure that there are no other metering installations before proceeding.
+====
+
+.Prerequisites
+
+*  The MeteringConfig custom resource is deleted.
```

Comment:
Do we need to specify a namespace for the `MeteringConfig` custom resource? That resource is namespaced-scoped and not cluster-scoped.

If so, you could probably get away with `The MeteringConfig custom resource in the openshift-metering namespace is deleted`.